website/docs/interactors/3-locators-filters-actions.md

```diff
@@ -98,8 +92,6 @@ Button('Submit').click();
 
 It’s that simple! Because Interactors can only match one element, there’s no ambiguity over which submit button you clicked. Additionally, the action will not occur if the Interactor is not there, so you don’t have to worry about checking if the button exists before issuing a click action.
 
-On the [Writing Interactors](/docs/interactors/write-your-own) page we'll also explain how you can construct your own actions to suit your needs when you create your own Interactors.
```

Comment:
I removed this line as it felt repetitive. We reference the `Writing Interactors` page earlier on the same page.